.github/workflows/test.yml

```diff
@@ -60,17 +62,41 @@ jobs:
 
       - run: conda info
 
+      - name: Install Julia on Windows
+        if: matrix.os == 'windows-latest'
+        run:
+          pip install julia
+
+      - name: Install Julia on Ubuntu/OSX
+        if: matrix.os != 'windows-latest'
+        run:
+          conda install -yq julia[version">=1.10.0"]
```

Comment:
These two lines are doing very different things:
1. `conda install julia` install the julia language.
2. `pip install julia` install the pyjulia python package.

It seems weird that this is necessary to specify this to make the test pass?

Reply:
`conda install julia` doesn't work on [Windows](https://discourse.julialang.org/t/how-to-request-for-conda-to-support-julia-install-for-windows-os/54092). I tried to find a channel that works but no luck.
I wasn't aware of the differences between the two installs. Do you think there's a workaround, or should we skip all julia-based tests altogether?

Reply:
There is a workaround :) (conda-forge/julia-feedstock#242)
We could try to add `-c -c https://repo.prefix.dev/julia-forge`. I already modified it in the julia helper.